packages/hoppscotch-cli/README.md

```diff
@@ -24,13 +24,24 @@ hopp [options or commands] arguments
 
    - Displays the help text
 
-3. #### **`hopp test <file_path>`**
+3. #### **`hopp test [options] <file_path>`**
    - Interactive CLI to accept Hoppscotch collection JSON path
    - Parses the collection JSON and executes each requests
    - Executes pre-request script.
    - Outputs the response of each request.
    - Executes and outputs test-script response.
 
+    #### Options:
+    ##### `hopp test -e / --env <file_path> test <file_path>`
```

Comment:
Since it is marked under the test section, you don't need to repeat the entire command
```suggestion
    ##### `-e <file_path>` / `--env <file_path>`
```